.github/workflows/npm-test.yml

```diff
@@ -0,0 +1,33 @@
+name: unit tests
+
+on: [push, pull_request]
+
+jobs:
+  run-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
```

Comment:
I think that in the future it will be possible for us to test on windows and drop browserstack entirely. Not that I hate browserstack but it would allow us to unify testing in pull requests and pushes. Just have to drop ie 11 support...